### PR TITLE
[LLM Runtime]update README and requirements

### PR DESCRIPTION
**Pull request by intellinjun**
## Type of Change

feature or bug fix or documentation or others
API changed or not
not
## Description
update graph README and requirements.txt
detail description 
JIRA ticket: xxx

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

**Comment by kevinintel** [comment]
please add the step of install neural speed.
remove unnecessary code snippet  in this doc

**Review by zhenwei-intel — APPROVED** [review]

**Comment by zhenwei-intel** [comment]
> please add the step of install neural speed. remove unnecessary code snippet in this doc.

add neural speed in requirement.txt

File: intel_extension_for_transformers/llm/runtime/graph/README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+
+> LLM Runtime has been renamed as **Neural Speed** and seperated as individual project, see [here](https://github.com/intel/neural-speed/tree/main) for more details.
+
+
 # LLM Runtime
```
Comment (kevinintel) [inline intel_extension_for_transformers/llm/runtime/graph/README.md:5]: Neural Speed

**Comment by VincyZhang** [comment]
need to remove `git+https://github.com/intel/neural-speed.git` in requirements.txt, but need to add how to build neural speed from source in README.md, please refer here: 
https://github.com/intel/intel-extension-for-transformers/blob/main/.github/workflows/script/unitTest/run_unit_test_llmruntime.sh#L35-L39

**Review by a32543254 — APPROVED** [review]
LGTM

File: intel_extension_for_transformers/llm/runtime/graph/requirements.txt [hunk 2]
```diff
@@ -10,3 +10,6 @@ peft
 datasets
 transformers_stream_generator
 tiktoken
```
Comment (a32543254) [inline intel_extension_for_transformers/llm/runtime/graph/requirements.txt:12]: add auto-gptq here

**Review by a32543254 — APPROVED** [review]
LGTM

**Review by airMeng — APPROVED** [review]

**Pull request merged**